.env.example

```
@@ -130,8 +130,8 @@ AUTOMATIC_OPENAI_STT_PROMPT="Transcribe Indian languages accurately. Recognize a
 DEEPGRAM_API_KEY=  # Your Deepgram API key
 DEEPGRAM_MODEL="nova-3-general"  # Deepgram model (nova-3-general recommended)
 DEEPGRAM_LANGUAGE="en"  # Single language for transcription (used when multi-language options are disabled)
-DEEPGRAM_ENDPOINTING=true  # Smart turn detection
-DEEPGRAM_VAD_EVENTS=true  # Voice activity detection events
+DEEPGRAM_ENDPOINTING=true  # Smart turn detection (true = default ~480ms, or integer ms e.g. 25)
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEEPGRAM_ENDPOINTING key should go before the DEEPGRAM_MODEL key

(UnorderedKey)